scripts/python_scripts/generate_network_status.py

```diff
@@ -147,19 +148,17 @@ def has_rewards_history(chain):
 			return True
 		return False
 
-	features = []
-
 	dict = {
-		"📚 Transfer History" : has_transfer_history,
-		"✨ ORML/Assets" : has_orml_or_asset,
+		"📚 Operation History" : has_transfer_history,
+		"✨ Multi assets" : has_orml_or_asset,
 		"📈 Staking analytics" : has_staking_analytics,
 		"🥞 Staking rewards": has_rewards_history
 	}
 
-	for key, value in dict.items():
-		for chain in chains:
-			if (value(chain)):
-				features.append(key)
+	features = []
+	for chain in chains:
+		features = [feature for feature, criteria in dict.items() if criteria(chain) == True]
```

Comment:
It seems like it is gonna override `features` for each chain and only the last chain will be included. Is that an expected behavior?

Reply:
It's expected, as on this line may be returned Nothing:
https://github.com/stepanLav/subquery-nova/blob/3f0f5c8a073dacb214d3c2379c7c771107aa0bd0/scripts/python_scripts/generate_network_status.py#L123

but, it looks not clear, so I changed implementation to this:
https://github.com/nova-wallet/subquery-nova/pull/138/commits/ac3692290e0804ae710973eb88806e6265a3b487